features/datum.feature

```diff
@@ -310,106 +308,99 @@ Functionaliteit: leveren van een datum
+        | <veld>.datum       | <datum>       |
+        | <veld>.jaar        | <jaar>        |
+        | <veld>.maand       | <maand>       |
+        | <veld>.onbekend    | <onbekend>    |
+        | <veld>.langFormaat | <langFormaat> |
 
       Voorbeelden:
-      | relatie | field    | groep                       | veld  | type           | GbaDatum | datum      | jaar | maand | onbekend | langFormaat  |
-      | ouder   | ouders   | geboorte                    | datum | Datum          | 20200308 | 2020-03-08 |      |       |          | 8 maart 2020 |
-      | ouder   | ouders   | geboorte                    | datum | DatumOnbekend  | 00000000 |            |      |       | true     | onbekend     |
-      | ouder   | ouders   | geboorte                    | datum | JaarDatum      | 20200000 |            | 2020 |       |          | 2020         |
-      | ouder   | ouders   | geboorte                    | datum | JaarMaandDatum | 20200300 |            | 2020 | 3     |          | maart 2020   |
-      | partner | partners | geboorte                    | datum | Datum          | 20200308 | 2020-03-08 |      |       |          | 8 maart 2020 |
-      | partner | partners | geboorte                    | datum | DatumOnbekend  | 00000000 |            |      |       | true     | onbekend     |
-      | partner | partners | geboorte                    | datum | JaarDatum      | 20200000 |            | 2020 |       |          | 2020         |
-      | partner | partners | geboorte                    | datum | JaarMaandDatum | 20200300 |            | 2020 | 3     |          | maart 2020   |
-      | partner | partners | aangaanHuwelijkPartnerschap | datum | Datum          | 20200308 | 2020-03-08 |      |       |          | 8 maart 2020 |
-      | partner | partners | aangaanHuwelijkPartnerschap | datum | DatumOnbekend  | 00000000 |            |      |       | true     | onbekend     |
-      | partner | partners | aangaanHuwelijkPartnerschap | datum | JaarDatum      | 20200000 |            | 2020 |       |          | 2020         |
-      | partner | partners | aangaanHuwelijkPartnerschap | datum | JaarMaandDatum | 20200300 |            | 2020 | 3     |          | maart 2020   |
-      | kind    | kinderen | geboorte                    | datum | Datum          | 20200308 | 2020-03-08 |      |       |          | 8 maart 2020 |
-      | kind    | kinderen | geboorte                    | datum | DatumOnbekend  | 00000000 |            |      |       | true     | onbekend     |
-      | kind    | kinderen | geboorte                    | datum | JaarDatum      | 20200000 |            | 2020 |       |          | 2020         |
-      | kind    | kinderen | geboorte                    | datum | JaarMaandDatum | 20200300 |            | 2020 | 3     |          | maart 2020   |
+      | relatie   | danrelatie | field    | groep                       | veld  | type           | GbaDatum | datum      | jaar | maand | onbekend | langFormaat  | Lo3-naam              |
```

Comment:
kolom 5 van deze tabel kan verwijderd worden. De waarde kan hard in het scenario geplaatst worden aangezien deze voor alle voorbeelden gelijk is.